include/aspect/utilities.h

```diff
@@ -130,7 +130,15 @@ namespace aspect
                                            const double semi_major_axis_a,
                                            const double eccentricity);
 
-
+      /**
+       * This enum lists available coordinate systems that can be used for 
+       * the function variables. Allowed values are 'cartesian', 
+       * 'spherical', and 'depth'. 'spherical' coordinates follow: r, phi 
+       * (2D) or r, phi, theta (3D); where r is radius, phi is longitude,
+       * and theta is the polar angle (Colatitude). 'depth' will create a 
+       * function, in which only the first parameter is non-zero, which is 
+       * interpreted to be the depth of the point.
```

Comment:
maybe write this as "The 'depth' coordinate system is a one-dimensional coordinate system in which only the depth, i.e., the distance below the 'top' surface as defined by each geometry model, is used."